editor/components/post-publish-button/test/index.js

```diff
@@ -2,67 +2,41 @@
  * External dependencies
  */
 import { shallow } from 'enzyme';
-import { merge } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import { PostPublishButton } from '../';
 
 describe( 'PostPublishButton', () => {
-	const user = {
-		data: {
-			id: 1,
-			post_type_capabilities: {
-				publish_posts: true,
-			},
-		},
-	};
-
-	const contributor = merge( {}, user, {
-		data: {
-			post_type_capabilities: {
-				publish_posts: false,
-			},
-		},
-	} );
-
 	describe( 'disabled', () => {
-		it( 'should be disabled if current user is unknown', () => {
-			const wrapper = shallow(
-				<PostPublishButton user={ {} } />
-			);
-
-			expect( wrapper.prop( 'disabled' ) ).toBe( true );
-		} );
-
 		it( 'should be disabled if post is currently saving', () => {
 			const wrapper = shallow(
-				<PostPublishButton user={ user } isSaving />
+				<PostPublishButton hasPublishAction={ true } isSaving />
```

Comment:
`hasPublishAction` - should it default to `true` when not provided?

Reply:
> should it default to `true` when not provided?

I'm not sure. My concern would be the possibility of showing incorrect UI to the end user. Could you explain your thinking?